frontend/www/js/omegaup/components/login/Signup.vue

```diff
@@ -79,10 +79,10 @@
             ></vue-recaptcha>
           </div>
           <div class="col-md-4 col-md-offset-6">
-            <div class="form-group">
+            <div class="form-group introjsregister">
               <button
                 data-signup-submit
-                class="btn btn-primary form-control"
+                class="btn btn-primary form-control introjsclass"
```

Comment:
Parece que `introjsclass` no se utiliza en ningún lugar, así que se puede quitar:
```suggestion
                class="btn btn-primary form-control"
```